plugins/inputs/logfile/logfile.go

```diff
@@ -259,6 +266,10 @@ func (t *LogFile) FindLogSrc() []logs.LogSrc {
 		}
 	}
 
+	if allowedOpenFileLimit, err := fdlimit.CurrentOpenFileLimit(); err == nil {
+		t.Log.Infof("Number of current monitored log files / Allowed monitored log files: %v/%v", t.tailerQueue.Size(), allowedOpenFileLimit)
```

Comment:
Realistically, we aren't tracking the total "allowable" number of log files to monitor. If they run out of file descriptors, we can tail the file, maybe, but can't do anything with it. 

I think this could be reworded:
```suggestion
		t.Log.Infof("Number of fds used by agent / Total allotted fds: %v/%v", t.tailerQueue.Size(), allowedOpenFileLimit)
```